app/src/main/java/org/zephyrsoft/trackworktime/EventListActivity.java

```diff
@@ -384,6 +399,18 @@ private String formatType(TypeEnum type) {
 						throw new IllegalStateException("unrecognized event type");
 				}
 			}
+
+			private String getTaskName(Integer taskId) {
+				if (taskId == null) {
+					return null;
+				}
+				Task task = taskIdToTaskMap.get(taskId);
+				if (task == null) {
+					Logger.error("No task for id: " + taskId);
```

Comment:
Although the effect won't be measurable because this probably won't ever happen (the user can't delete tasks which are still used in events), I'd generally prefer the SLF4J notation without string concatenation in log statements:

`Logger.error("No task for id: {}", taskId);`

In 282005a3 I went through the entire codebase and eliminated any manual string concatenation for logging, I'd like to keep it this way... :blush:

Reply:
Sure, not a problem at all